### PR TITLE
Add adkris1002 page

### PR DESCRIPTION
**Pull request by Adkris1002**
@leewujung I have made my web page and I am tagging you to merge my branch! Take a look and let me know by Slack if I need to change anything :)

### CODE REVIEW

File: content/authors/adkris/_index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Researchers
+---
+
+I am a student at the University of Washington majoring in Electrical Engineering with a concentration in Digital Signal Processing. I will be graduating around the Spring of 2024 and look forward to pursuing graduate studies after receiving my bachelor's degree.
```
Comment (leewujung) [inline content/authors/adkris/_index.md:69]: suggested change:
"I will be graduating around the Spring of 2024 ..." --> "I am expected to graduate ..."
and corresponding grammar change in the second part of the sentence.

File: content/authors/adkris/_index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Researchers
+---
+
+I am a student at the University of Washington majoring in Electrical Engineering with a concentration in Digital Signal Processing. I will be graduating around the Spring of 2024 and look forward to pursuing graduate studies after receiving my bachelor's degree. 
+
+I have experience in Java, Python, and Arduino. I am interested in using my programming skills to aid wildlife conservation efforts. My current project with Echospace involves monitoring the bat population at the Union Bay Natural Area, beside the Husky Stadium, using their echolocation call activity to develop machine learning algorithms that generate meaningful information on the overall population and ecosystem.
```
Comment (leewujung) [inline content/authors/adkris/_index.md:71]: ", beside" the Husky Stadium --> ", which is next to " the Husky Stadium

Comment (leewujung) [reply content/authors/adkris/_index.md:71]: also, it is likely overstatement to say that you would derive information about "the overall population and ecosystem" based on the echolocation calls. There are things that we can infer and make interpretation for, but I wouldn't over sale it at this stage. Try changing it to something more local and behavior-oriented?

**Comment by leewujung** [comment]
@Adkris1002 : thanks for the PR! I've made a few suggestions and you should have gotten the email notifications. 

This procedure is also meant to be a way to make sure you're familiar with the GitHub PR workflow for coding collaboratively, so I won't be using slack here. :) 

I also noticed that you used "Add file via upload" to put info to your fork. If you haven't used git local vs remote setup, I would strongly recommend that you practice that in this PR (i.e., you will be pushing up your commit from a local command line or desktop app interface instead of adding file or editing directly on web browser).

File: content/authors/adkris/_index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+bio: I am interested in applying artificial intelligence for environmental conservation
+
+interests:
+- Bioacoustics
+- Wildlife conservation
+- Artificial Intelligence 
+
+education:
+  courses:
+  - course: BE in Electrical Engineering (in progress)
```
Comment (leewujung) [inline content/authors/adkris/_index.md:33]: It is often B"S" in EE so you probably wanna check that.

**Comment by Adkris1002** [comment]
@leewujung Thank you for all the feedback! I have started to now use the command line to add-commit-push, I believe I resolved all your suggestions but feel free to take a look again!

File: content/authors/adkris/_index.md [hunk 4]
```diff
@@ -66,9 +66,9 @@ user_groups:
 - Researchers
 ---
 
-I am a student at the University of Washington majoring in Electrical Engineering with a concentration in Digital Signal Processing. I will be graduating around the Spring of 2024 and look forward to pursuing graduate studies after receiving my bachelor's degree. 
+I am a student at the University of Washington majoring in Electrical Engineering with a concentration in Digital Signal Processing. I am expected to graduate around the Spring of 2024 and look forward to hope to pursue graduate studies after receiving my bachelor's degree.
```
Comment (leewujung) [inline content/authors/adkris/_index.md:69]: look forward to hope to pursue --> look forward to pursuing

**Comment by leewujung** [comment]
Looks great! I made a small grammar suggestion and otherwise this is ready to be merged. :)

**Comment by Adkris1002** [comment]
Totally missed that typo! Thank you!

**Comment by leewujung** [comment]
Great, thanks for making the changes. Merging now 🚀

**Pull request merged**